### ISSUE TITLE
[IOS] Some strange issues 

### ISSUE THREAD

**Issue by ishigamii** [react-native-kit/react-native-track-player#314]
Hi, I am currently using the version 1.0.0 of the library and first of all that's an amazing job you are doing there. After some struggle (sadly can't make a PR for the wiki or don't know how cause documentation is laking a little for more details like jump forward and other things) I got a very nice render of what I wanted. 

**Configuration**
package.json
```
    "react": "16.3.1",
    "react-native": "0.55.4",
    "react-native-track-player": "^1.0.0",
```

Though I am having two issues on iOS (I didn't try the lib so far on Android) : 

- Jump forward / Jump backward 

I am initialising my player like follow : 
```
  static async initPlayer() {
    await TrackPlayer.setupPlayer();
    TrackPlayer.updateOptions({
      jumpInterval: 60,
      capabilities: [
        TrackPlayer.CAPABILITY_PLAY,
        TrackPlayer.CAPABILITY_PAUSE,
        TrackPlayer.CAPABILITY_SEEK_TO,
        TrackPlayer.CAPABILITY_JUMP_FORWARD,
        TrackPlayer.CAPABILITY_JUMP_BACKWARD,
      ],
    });
  }
```
But for this configuration I happened to have all of this results (I think the output changes considering when I open the media center, if I open it very fast before/during the track adding it usually bugs and give me a missing side or both side missing)

Normal output : 
<img width="366" alt="capture d ecran 2018-10-24 a 17 17 58" src="https://user-images.githubusercontent.com/26909067/47441627-d6a2a780-d7b0-11e8-81f1-c76d6ef92148.png">

Left missing : 
<img width="368" alt="capture d ecran 2018-10-24 a 17 13 05" src="https://user-images.githubusercontent.com/26909067/47441625-d6a2a780-d7b0-11e8-94c5-9c9800fb5f20.png">

Both missing : (I think it was cause the song wasn't loaded yet
<img width="363" alt="capture d ecran 2018-10-24 a 17 17 29" src="https://user-images.githubusercontent.com/26909067/47441626-d6a2a780-d7b0-11e8-9122-03c012d436ac.png">

 - Multiple events fired 

For some reason, after a while sometimes (it doesn't always happens) the app starts to fire the event multiple times instead of once. Sometimes up to 4 times each like if I had lots of listeners for the same event even though I did declare it once. In addition to that the player start to behave strangely (because of this I think) cause the pause is pausing and the play is restarting the file (I receive the `playing` event but also the `buffering` event like if he was playing a completely new file) 
I don't have any way to reproduce the issue so far except waiting while the song is played and try the button after a while

My configuration is as described in the documentation : 
in my `index.js` where I have my `AppRegistry.registerComponent` I added `TrackPlayer.registerEventHandler(require('./player-handler.js'));`

this one being : 
```
import TrackPlayer from 'react-native-track-player';

module.exports = async (data) => {
  console.log('la', data.type, data.position, data);
  if (data.type === 'playback-state') {
    // Update the UI with the new state
  } else if (data.type === 'remote-play') {
    TrackPlayer.play();
  } else if (data.type === 'remote-pause') {
    TrackPlayer.pause();
  } else if (data.type === 'remote-seek') {
    TrackPlayer.seekTo(data.position);
  } else if (data.type === 'remote-jump-backward') {
    TrackPlayer.getPosition().then((position) => {
      TrackPlayer.seekTo(position - data.interval);
    });
  } else if (data.type === 'remote-jump-forward') {
    TrackPlayer.getPosition().then((position) => {
      TrackPlayer.seekTo(position + data.interval);
    });
  } else if (data.type === 'playback-queue-ended') {
    TrackPlayer.reset();
  }
};

```

**Comment by curiousdustin** [comment]
I am using very similar capabilities options, and I also will sometimes see things like the "Left Missing" example above. I haven't yet figured out how to reproduce this behavior consistently.

**Comment by ishigamii** [comment]
Currently having the same problems with the **dev branch** except for the `Multiple events fired` that I can't reproduce for the moment.

**Comment by ishigamii** [comment]
Closed because of the new version `1.1.2`. I will open a new issue with android / iOS bugs
